app/services/cancel_referee.rb

```diff
@@ -1,6 +1,6 @@
 class CancelReferee
   def call(reference:)
-    reference.update!(feedback_status: 'cancelled')
+    reference.reload.update!(feedback_status: 'cancelled')
```

Comment:
Not sure if this feels a bit wrong? Bullet being bullet i guess

Reply:
Yeah I think you're right. 
The Bullet gem is raising an `eager loading` error on the spec when it comes to cancelling a reference. Both @malcolmbaig  and I have tried to investigate the cause of this to no avail -  we think it's possibly a false positive. Reloading makes the Bullet error disappear but then probably not so good we are modifying source code in this way for the sake of a test. The alternative is to disable bullet in the spec concerned and add a comment detailing findings from our investigation. What do you think?

Reply:
Yea I had to something similar in this spec candidate_interface/offers_and_withdrawls/candidate_withdraws_spec.rb a while ago.

![image](https://user-images.githubusercontent.com/42515961/101341435-fd30d900-3878-11eb-9f28-0cc81bf763a7.png)

I think not touching source code is preferable.

Reply:
Cool - I've disabled Bullet in the spec